### PR TITLE
Update links to samples, #28106

### PR DESCRIPTION
**Pull request by patriknw**
Started to update links to samples to see if this is the right approach.
* adding Example project section at the end of corresponding "main" doc page
* new examples.md page to collect links to all at one place

Refs #28106

### CODE REVIEW

**Comment by patriknw** [comment]
updated with remaining

File: akka-docs/src/main/paradox/typed/cluster-sharding.md [hunk 1]
```diff
@@ -410,3 +410,11 @@ One important configuration property is `number-of-shards` as described in @ref:
...
 
 @@snip [reference.conf](/akka-cluster-sharding-typed/src/main/resources/reference.conf) { #sharding-ext-config }
+
+## Example project
+
+@java[@extref[Sharding example project](samples:akka-sample-sharding-java)]
+@scala[@extref[Sharding example project](samples:akka-sample-sharding-scala)]
+is an example project that can be downloaded, and with instructions of how to run.
+
+This project contains a KillrWeather sample illustrating how to use Cluster Sharding.
```
Comment (helena) [inline akka-docs/src/main/paradox/typed/cluster-sharding.md]: Nice!

**Comment by helena** [comment]
Overall I like it, those repetitive `You can look at the...` or `The easiest way to...` phrases and their wording somehow always stuck out as odd.

That said I want to be sure it is easy for users to find the examples.

**Comment by helena** [comment]
Can we add a link/links to any other indexes for the examples.md? It seems obscure if we only have it in that one index.

**Comment by patriknw** [comment]
> Can we add a link/links to any other indexes for the examples.md? It seems obscure if we only have it in that one index.

Don't know where it would be?

Seems like there was no uptake on the suggestion to have the `Example project` section at the top of the pages, which I can understand. Therefore I placed it at the end of the pages.

I don't mind adding more links at specific places in the content text to relevant sample.

**Review by chbatey — APPROVED** [review]
Looks good

**Review by johanandren — APPROVED** [review]

**Comment by helena** [comment]
@patriknw Perhaps the additional awareness of the new examples should come from a common slide we all show in our akka videos.

**Review by helena — APPROVED** [review]

**Comment by patriknw** [comment]
yes, we can promote them in various ways. tweeting about specific samples is another way. Since some of them are massively updated that deserves to be known.

File: akka-docs/src/main/paradox/multi-jvm-testing.md [hunk 2]
```diff
@@ -210,3 +208,10 @@ the sbt prompt.
 There has also been some additions made to the `SbtMultiJvm` plugin to accommodate the
 @ref:[may change](common/may-change.md) module @ref:[multi node testing](multi-node-testing.md),
 described in that section.
+
+## Example project
+
+@extref[Multi JVM example project](samples:akka-sample-multi-node-scala)
+is an example project that can be downloaded, and with instructions of how to run.
```
[the remark this answers is not in the document]
Comment (patriknw) [reply akka-docs/src/main/paradox/multi-jvm-testing.md]: fixed

**Pull request merged**